src/helm-frontend/src/services/getSchema.ts

```diff
@@ -6,10 +6,12 @@ import getBenchmarkRelease from "@/utils/getBenchmarkRelease";
 
 export default async function getSchema(signal: AbortSignal): Promise<Schema> {
   try {
+    // TODO: this should not be hard-coded to the main website, want to be able
```

Comment:
This is the wrong URL because the schema file is now under `benchmark_output`, not under the static directory. Follow this [example](https://github.com/stanford-crfm/helm/blob/b49e75c3fda3bd08cf03aafeeb058012a373b2b3/src/helm-frontend/src/services/getGroupsTables.ts#L5-L7) to get the right URL:

```javascript
import getBenchmarkEndpoint from "@/utils/getBenchmarkEndpoint";
import getVersionBaseUrl from "@/utils/getVersionBaseUrl";

export function getSchemaJsonUrl(): string {
  return getBenchmarkEndpoint(`${getVersionBaseUrl()}/schema.json`);
}
```

Reply:
Ah, right.  Replaced with this code.  Although when I run the local React server, it doesn't seem to resolve to the right URL...